src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const priceForDay = 40;
+  const discountOne = 20;
+  const discountTwo = 50;
+  const totalCost = days * priceForDay;
+
+  if (days < 3) {
+    return totalCost;
+  }
+
+  if (days >= 3 && days < 7) {
+    return totalCost - discountOne;
+  }
+
+  if (days >= 7) {
+    return totalCost - discountTwo;
+  }
```

Comment:
minor fix - but cleaner way
```suggestion
    return totalCost - discountTwo;
```
No need to check for the condition, as it is always true